sdk/keyvault/ci.yml

```diff
@@ -21,18 +26,22 @@ pr:
   branches:
     include:
     - master
+    - feature/*
+    - hotfix/*
+    - release/*
   paths:
     include:
     - sdk/keyvault/
 
-resources:
-  repositories:
-    - repository: azure-sdk-tools
-      type: github
-      name: Azure/azure-sdk-tools
-      endpoint: azure
-
-jobs:
-- template: ../../eng/pipelines/templates/jobs/archetype-sdk-client.yml
+stages:
+- template: ../../eng/pipelines/templates/stages/archetype-sdk-client.yml
   parameters:
     ServiceDirectory: keyvault
+    ArtifactName: packages
+    Artifacts:
+    - name: Azure.Security.KeyVault.Certificates
+      safeName: AzureSecurityKeyVaultCertificates
+    - name: Azure.Security.KeyVault.Keys
+      safeName: AzureSecurityKeyVaultKeys
+    - name: Azure.Security.KeyVault.Secrets
+      safeName: AzureSecurityKeyVaultSecrets
```

Comment:
Nit: the .editorconfig dictates to add new lines to EOF, though probably doesn't cover .yml. Would be nice to keep a new line at the end here to help with diff'ing/merging in the future.

Reply:
Fixed.